index.html

```diff
@@ -163,10 +163,33 @@ <h3>Multi-Post Stories<br>Gain+Glory</h3>
                     <li>JavaScript</li>
                     <li>html</li>
                 </ul>
-                <button onclick="location.href='#';">See Project</button>
+                <button onclick="revealDetails(5)">See Project</button>
             </div>
         </div>
     </section>
+    <div id="portfolio-details">
+        <img id="detailImg" src="" alt="">
+        <h1 id="detailH1"></h1>
+        <ul id="detailUl"></ul>
+        <p id="detailP"></p>
+        <div class="buttons">
+            <button>
+                See Live
+                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ ] Good job on the popup, Yet compared with the main design one thing missing is unlike the Figma design the background content is clearly visible on your design.

## Figma design 
![Screenshot 2023-01-07 193152](https://user-images.githubusercontent.com/69077061/211161056-0d309f4b-8a89-44a0-a4c1-48ee28c5d9e4.png)
## Your design.
![Screenshot 2023-01-07 193228](https://user-images.githubusercontent.com/69077061/211161059-529e13a1-2b52-444c-8f6c-0f911cb15b11.png)

- [ ] Also, on a screen size around 770px to 1000px your popup seems to be overflowing, Pleas address this for a better responsive design. 

![Screenshot 2023-01-07 193729](https://user-images.githubusercontent.com/69077061/211161167-3df0d24b-b7a2-47b5-bc11-ad6fbb302e1e.png)